internal/utils/misc.go

```diff
@@ -40,7 +40,7 @@ const (
 	VectorImage      = "timberio/vector:0.28.1-alpine"
 	PgbouncerImage   = "bitnami/pgbouncer:1.20.1-debian-11-r39"
 	PgProveImage     = "supabase/pg_prove:3.36"
-	GotrueImage      = "supabase/gotrue:v2.145.0"
+	GotrueImage      = "supabase/gotrue:v2.147.1"
```

Comment:
This isn't out yet, but will be deployed soon. Currently, we are on v2.147.0 on middleware and v2.146 for all projects. Can wait to bump this too if that's better.